apps/web/src/design-system/switch/Switch.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { Switch as MantineSwitch, SwitchProps } from '@mantine/core';
+import useStyles from './Switch.styles';
+
+interface ISwitchProps {
+  label?: React.ReactNode;
+  checked?: boolean;
+}
+
+/**
+ * Switch component
+ *
+ */
+export function Switch({ ...props }: ISwitchProps) {
+  const { classes } = useStyles();
+  const defaultDesign = { radius: 'xl', size: 'md' } as SwitchProps;
+  const [checked, setChecked] = useState(false);
+  return (
+    <MantineSwitch
+      classNames={classes}
+      {...defaultDesign}
+      checked={checked}
+      onChange={(event) => setChecked(event.currentTarget.checked)}
```

Comment:
When we will start to integrate forms we will need to add a callback to onChange. 
I think that may be the right approach is to remove the checked state because it will be overridden by the ...props restructuring.

And if the parent passed the checked value it will be his responsibility to watch the onChange event, and if `checked` is undefined, than it should work as expected. wdyt?